doc/pages/Manual.md

```diff
@@ -1294,7 +1298,9 @@ will be copied over the root of the package source. If already present, files
 are overwritten, and directories are recursively merged. [`opam`](#opam) file
 fields like [`patches:`](#opamfield-patches) refer to files at that same root,
 so patches specific to opam are typically included in
-this subdirectory.
+this subdirectory. Since opam 2.3, the [`extra-files:`](#opamfield-extra-files)
+must be list each files in this directory. Before opam 2.3 this field was
+optional.
```

Comment:
```suggestion
this subdirectory. Since opam 2.3, files must be listed in the
[`extra-files:`](#opamfield-extra-files) or they are ignored. Before
opam 2.3, all files were copied regardless of whether they appeared
in the `extra-files` list.
```